File: Album_Store_BackEnd/AlbumStore/AlbumStore.Domain/Entities/CollectionItem.cs
namespace AlbumStore.Domain.Entities;
public class CollectionItem
{
    public virtual Guid Id { get; set; }
    public virtual string UserId { get; set; }
    public virtual ApplicationUser User { get; set; }
    public virtual Guid? ProductId { get; set; }
    public virtual Product Product { get; set; }
    public virtual Guid? ImageId { get; set; }
    public virtual Image Image{ get; set; }
    public string Title { get; set; }
    public string Artist { get; set; }
}

File: AlbumStoreUI/app/src/main/java/com/albumstore/todo/data/collection/CollectionItem.kt
package com.albumstore.todo.data.collection

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.albumstore.todo.data.product.ImageDto

@Entity(tableName = "collection_items")
class CollectionItem (
    @PrimaryKey
    var id: String = "",
    val productId: String? = "",
    var imageId: String? = "",
    val title: String = "",
    val artist: String = "",
    val image: ImageDto? = null,
) {
    override fun toString(): String {
        return "CollectionItem(id='$id', productId=$productId, imageId=$imageId, title='$title', artist='$artist', imageBase64=${image?.imageBase64?.take(10)}" +
                ", contentType=${image?.contentType}, fileName=${image?.fileName})"
    }
}
